app/serializers/person_escort_record_serializer.rb

```diff
@@ -0,0 +1,25 @@
+class PersonEscortRecordSerializer < ActiveModel::Serializer
+  belongs_to :profile, record_type: :profile
+  belongs_to :framework, record_type: :framework
+  has_many :framework_responses, serializer: FrameworkResponseSerializer, key: :responses
+
+  attributes :version, :status
+
+  def status
+    object.state
+  end
+
+  def version
+    object.framework.version
```

Comment:
Would prefer to add this delegation in the `PersonEscortRecord` model instead

Reply:
This is a temporary implementation till we finalise the way we accept/persist versions. In the future this will be persisted so didn't want to add too much before we flushed that out soon and just wanted to return a value for now

Reply:
Fair enough 👍